packages/devtools_app/lib/src/inspector/inspector_service.dart

```diff
@@ -555,7 +561,13 @@ class ObjectGroup {
     String methodName,
     Map<String, Object> params,
   ) {
-    return _callServiceExtension('ext.flutter.inspector.$methodName', params);
+    final callMethodName = 'ext.flutter.inspector.$methodName';
+    if (serviceManager.serviceExtensionManager
```

Comment:
should this be if _not_ available? like above in `invokeServiceMethodDaemonNoGroup`?

Reply:
Yes it should.  I caught it by debugging the problem - good eyes.